charts/pihole/.helmignore

```diff
@@ -19,3 +19,9 @@
 .project
 .idea/
 *.tmproj
+
+# Manually added entries
+ci/
+examples/
+Makefile
+README.md.gotmpl
```

Comment:
Anything that is in the folder next to Chart.yaml will be packaged into the .tgz file the Helm chart is distributed as, and then things in turn will be installed and saved for use in a k8s Secret with each installed version of the Helm chart etc.

I ended up learning about the need for a Helmignore before when a file I had (in a separate Helm chart) were so large that I couldn't fit it inside the k8s Secret and Helm errored when trying to install a Helm chart.